### ISSUE TITLE
How to get the performance due to parameter hypers['cnt']?

### ISSUE THREAD

**Issue by mrsempress** [lxcnju/FedRepo#4]
I only retrain FedAVG with default parameters (`hypers['cnt']=2`, C100-100-20, VGG11). And the results are $46.48+0.6703$ and $47.01+1.7480$, which is lower than the results reported in your paper: $49.8+0.471$.
Question 1: Since there are two results, the final result is the average value or the maximum value. 
Question 2: The overall results are lower than those reported in the paper.

**Comment by mrsempress** [comment]
I experimented again, and the result was $46.92+3.6956$ and $47.16+3.8267$. The performance is still low, and the variance value fluctuates wildly.

**Comment by lxcnju** [comment]
The best result is reported under different hyper-parameters. The learning rate matters when using VGG11 on CIFAR-100. Set LR as 0.01, 0.05, or 0.03.
